.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,19 +6,24 @@ If unsure about something, just do as best as you're able.
 If your PR only contains changes to documentation, you may skip the template below.
 -->
 
-### Pull Request check-list
+### Pull Request Checklist
 
 _Please make sure to review and check all of these items:_
 
-- [ ] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
-- [ ] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
 - [ ] Have you added new tests to prevent regressions?
+- [ ] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
 - [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
 - [ ] Did you update the typescript typings accordingly (if applicable)?
+- [ ] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
 - [ ] Did you follow the commit message conventions explained in [CONTRIBUTING.md](https://github.com/sequelize/sequelize/blob/main/CONTRIBUTING.md)?
 
 <!-- NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open. -->
 
-### Description of change
+### Description Of Change
 
 <!-- Please provide a description of the change here. -->
+
+### Todos
```

Comment:
What do you expect here?

Reply:
User can specify the todos in the pull requests instead of checking important check list, it might be easier for reviewers to judge Is it true that what he did (besides commit) ensures that he has done things according to the issue and important checklist.

Reply:
Should we have a comment about what this can be used for or an example maybe?

Reply:
Example in a comment that is?